var/spack/repos/builtin/packages/ffmpeg/package.py

```diff
@@ -28,4 +33,7 @@ def configure_args(self):
         if '+shared' in spec:
             config_args.append('--enable-shared')
 
+        if '+aom' in spec:
+            config_args.append('--enable-libaom')
```

Comment:
Can you explicitly add:
```python
else:
    config_args.append('--disable-libaom')
```
? Otherwise, Autotools sometimes likes to enable dependencies if it finds them, making the build non-deterministic.

Reply:
Cheers, done